techniques/system/inventory/1.0/fusionAgent.st

```diff
@@ -505,10 +505,6 @@ bundle agent addInformationsToInventory
     xen::
       "VMRUDDERUUID"     string => execresult("${VMRUDDERUUID_cmd}", "noshell");
 
-    # define polserv_uuid
-    any::
-      "polserv_uuid"   string => readfile( "${sys.workdir}/rudder-server-uuid.txt" , "50" );
```

Comment:
This values is already defined in doInventory_always by reading the same file.
And this bundle is called with the same condition as the file is read (uuid_succesfully_downloaded)